packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -298,15 +298,21 @@ export async function deployPreBuilt(
         if (previousSyncAndActionConfig) {
             bumpedVersion = increment(previousSyncAndActionConfig.version as string | number).toString();
 
-            const syncs = await getSyncsByProviderConfigAndSyncName(environment_id, provider_config_key, sync_name);
-            for (const sync of syncs) {
-                if (!runs) {
-                    continue;
-                }
-                const { success, error } = await updateSyncScheduleFrequency(sync.id as string, runs, sync_name, environment_id, activityLogId as number);
-
-                if (!success) {
-                    return { success, error, response: null };
+            if (runs) {
```

Comment:
Modified the condition because it seems `getSyncsByProviderConfigAndSyncName` was called for nothing in case there was no `runs`. Tell me if I need to revert the condition maybe I missed something ☺️